File: docs/examples/py/src/data_management_delete_data.py
from reduct import Client, Bucket


async def browse_buckets():
    # Create a client with the base URL and API token
    async with Client("http://localhost:8383", api_token="my-token") as client:
        # Get bucket to remove
        bucket: Bucket = await client.get_bucket("bucket-to-remove")

        # Delete only entry with name "example-entry"
        await bucket.remove_entry("example-entry")

        # Remove entire bucket
        await bucket.remove()

File: docs/examples/rs/examples/data_management_delete_data.rs
use reduct_rs::{ReductClient, ReductError};
use tokio;

#[tokio::main]
async fn main() -> Result<(), ReductError> {
    // Create a new client with the API URL and API token
    let client = ReductClient::builder()
        .url("http://127.0.0.1:8383")
        .api_token("my-token")
        .build();

    // Get bucket to remove
    let bucket = client.get_bucket("bucket-to-remove").await?;

    // Delete only entry with name "example-entry"
    bucket.remove_entry("example-entry").await?;

    // Remove entire bucket
    bucket.remove().await?;
    Ok(())
}
